command/project_set.go

```diff
@@ -66,6 +68,14 @@ func (cmd *ProjectSet) DoRun(args []string) (err error) {
 		return HandleError(errors.New("Project not found, please check the project name. You can get a list of your projects by running `nerd project list`."))
 	}
 
+	var p populator.P
+	p = &populator.EnvPopulator{}
```

Comment:
Maybe make this this configurable with a flag? like '--config-src=env' or something?